### PR TITLE
Added an include for logging articles regarding samples browser

### LINKED ISSUE #25314: Doc references a sample app that is hard to find or not linked
The doc references a sample app, but I can't find it anywhere. A link either inline or somewhere in a high level callout would be useful.

> The sample app creates a field called _processingWorkScope (Internal/LoggerExtensions.cs)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: aa00c051-c047-bbcd-5e25-ca6a722b51c3
* Version Independent ID: 3f5f1b6e-06ac-cc07-48cd-b8b957a4abbb
* Content: [High-performance logging in .NET](https://docs.microsoft.com/en-us/dotnet/core/extensions/high-performance-logging)
* Content Source: [docs/core/extensions/high-performance-logging.md](https://github.com/dotnet/docs/blob/main/docs/core/extensions/high-performance-logging.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @IEvangelist
* Microsoft Alias: **dapine**

### PR DESCRIPTION
**Pull request by IEvangelist**
## Summary

Added an include that can be plugged into logging articles with samples browser snippets

Fixes #25314, relies on https://github.com/dotnet/samples/pull/4719

Internal preview

✔️ [Logging in .NET](https://review.docs.microsoft.com/en-us/dotnet/core/extensions/logging?branch=pr-en-us-25402&tabs=command-line)

### CODE REVIEW

**Review by gewarren — APPROVED** [review]
LGTM.

**Review by adegeo — DISMISSED** [review]
LGTM!

File: docs/core/extensions/includes/logging-samples-browser.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+author: IEvangelist
+ms.author: dapine
+ms.date: 07/30/2021
+ms.topic: include
+---
```
Comment (Youssef1313) [inline docs/core/extensions/includes/logging-samples-browser.md:6]: I'm wondering if these metadata does anything for includes?

File: docs/core/extensions/includes/logging-samples-browser.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+> [!TIP]
+> All of the logging example source code is available in the **Samples Browser** for download. For more information, see [Browse code samples: Logging in .NET](/samples/dotnet/samples/csharp-logging-fundamentals).
```
Comment (Youssef1313) [inline docs/core/extensions/includes/logging-samples-browser.md:9]: Not really for download (currently):

Related issues: https://github.com/dotnet/samples/issues/4723 https://github.com/dotnet/docs/issues/23380

Comment (IEvangelist) [reply docs/core/extensions/includes/logging-samples-browser.md:9]: We are aware of this - and as @adegeo said here, https://github.com/dotnet/samples/issues/4723#issuecomment-890438052 it's being addressed internally soon.

File: docs/core/extensions/includes/logging-samples-browser.md (hunk 1 above)
Comment (IEvangelist) [reply docs/core/extensions/includes/logging-samples-browser.md:6]: From our internal contributor guides (we should really make these public):

image

Comment (Youssef1313) [reply docs/core/extensions/includes/logging-samples-browser.md:6]: > (we should really make these public):

Yess fully agree with that.

**Review by BillWagner — APPROVED** [review]
This LGTM. You can :shipit: when ready

**Pull request merged**